cleanup/binding-clean.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# set -x
+set -e
+
+CLEAR='\033[0m'
+RED='\033[0;31m'
+
+# Location of the yaml to use to deploy the cleanup job
+yaml_url=https://raw.githubusercontent.com/rancher/rancher/master/cleanup/binding-clean.yaml
```

Comment:
Is this assuming that we will merge this to master after this 2.5 PR is merged? otherwise this will 404

Reply:
Yup, master next.